extensions/amp-social-share/1.0/amp-social-share.css

```diff
@@ -20,6 +20,16 @@
   * See https://github.com/ampproject/amphtml/issues/4277 for more details.
   */
 
+amp-social-share:focus-within {
+  outline: #0389ff solid 2px;
+  outline-offset: 2px;
+}
+
+amp-social-share::part(button):focus {
+  outline: none;
```

Comment:
This makes sense. But we may need to add `!important`.

Reply:
I have added in latest revision.  FMI - this is to prevent the user from overwriting in his / her own amp page by accident?